apps/emqx_connector/src/emqx_connector_http.erl

```diff
@@ -606,21 +651,13 @@ is_sensitive_key([{str, StringKey}]) ->
 is_sensitive_key(Atom) when is_atom(Atom) ->
```

Comment:
with the string wrapped, do we still need this ?

Reply:
Yes, it's two different concepts, `redact`, which uses this, is called at various places where we want to log data and for most parts this happens either before or after wrapping/unwrapping - or even on raw data coming from the actual http client.